tests/rules/test_empty_values.py

```diff
@@ -62,6 +62,30 @@ def test_in_block_mappings_single_line(self):
                    'implicitly-null:with-colons:in-key2:\n', conf,
                    problem1=(2, 37))
 
+    def test_forbid_key_without_values_eof(self):
+        conf = ('empty-values: {forbid-in-block-mappings: false,\n'
+                '               forbid-in-flow-mappings: false,\n'
+                '               forbid-key-without-values-eof: true}\n')
+        self.check('---\n'
+                   'implicitly-null\n', conf, problem1=(2, 16))
+        self.check('---\n'
+                   'implicitly-null:\n', conf)
+        self.check('---\n'
+                   'explicitly-null: null\n', conf)
+        self.check('---\n'
+                   'implicitly-null:with-colons:in-key\n', conf,
+                   problem1=(2, 35))
+        self.check('---\n'
+                   'implicitly-null:with-colons:in-key:\n', conf)
+        self.check('---\n'
+                   'explicitly-null:with-colons:in-key: null\n', conf)
+        self.check('---\n'
+                   'implicitly-null:with-colons:in-key\n'
+                   'key: value\n', conf, problem1=(3, 4, 'syntax'))
+        self.check('---\n'
+                   'implicitly-null:with-colons:in-key:\n'
+                   'key: value\n', conf)
```

Comment:
1. If *true positive* checks are added, you must also add the same *true negative* ones (i.e. the same YAML, but with the new option disabled). Please see how other options do, to see the best place to add them.
2. It would be a good idea to add at least an example of a nested dict:
   ```yaml
   # inside a list, itself inside a surrounding dict:
   a:
     - key1:value1
   ```

Reply:
ok, i will add thoses checks